src/status_im/ui/screens/profile/contact/views.cljs

```diff
@@ -30,13 +30,13 @@
    [common/network-info]
    [common/separator]])
 
-(defn profile-actions [{:keys [pending? whisper-identity dapp?]} chat-id]
+(defn profile-actions [{:keys [new? pending? whisper-identity dapp?]} chat-id]
   [react/view action-button.styles/actions-list
-   (if pending?
+   (if (or pending? new?)
```

Comment:
This I am not very keen on, essentially for a given contact there are 3 possible states: already in contacts, not in contacts, or pending. I used a different flag to signal that as re-using pending? would have been a bit confusing.

Reply:
we don't need `new?`, we can just check if `pending?` is `true` or `nil`